src/OrbitGl/TrackManager.cpp

```diff
@@ -235,11 +236,12 @@ std::vector<ThreadTrack*> TrackManager::GetSortedThreadTracks() {
 
   // Tracks with instrumented timers appear first, ordered by descending order of timers.
   // The remaining tracks appear after, ordered by descending order of callstack events.
-  std::sort(sorted_tracks.begin(), sorted_tracks.end(),
-            [&num_events_by_track](ThreadTrack* a, ThreadTrack* b) {
-              return std::make_tuple(a->GetNumberOfTimers(), num_events_by_track[a]) >
-                     std::make_tuple(b->GetNumberOfTimers(), num_events_by_track[b]);
-            });
+  orbit_base::sort(
+      sorted_tracks.begin(), sorted_tracks.end(),
+      [&num_events_by_track](ThreadTrack* track) {
+        return std::make_tuple(track->GetNumberOfTimers(), num_events_by_track[track]);
+      },
+      std::greater<>{});
```

Comment:
It's unfortunate that we are adding a line :-)

Reply:
I've also been surprised by that. I guess, it's due to the longer qualified name `orbit_base::sort` vs `std::sort`.